assembl/graphql/schema.py

```diff
@@ -318,14 +323,26 @@ def is_type_of(cls, root, context, info):
                 'Received incompatible instance "{}".'
             ).format(root))
         # return isinstance(root, cls._meta.model)  # this was the original code
-        return type(root) == type(cls._meta.model)
+        return type(root) == cls._meta.model or type(root) == models.RootIdea
+
+
+    def resolve_parent_id(self, args, context, info):
+        parents = self.get_parents()
+        if not parents:
+            return None
+
+        return Node.to_global_id('Idea', parents[0].id)
 
     def resolve_posts(self, args, context, info):
         connection_type = info.return_type.graphene_type  # this is PostConnection
         model = connection_type._meta.node._meta.model  # this is models.PostUnion
-        query = self.get_related_posts_query(
+        related = self.get_related_posts_query(True)
```

Comment:
Because I want a simpler query here, I don't need the counters stuff for the authenticated user (for now). 
Look at the difference between as_clause and as_clause_base in assembl/models/path_utils.py